### ISSUE TITLE
Sent image is shown in original resolution, not the real one

### ISSUE THREAD

**Issue by waclaw66** [vector-im/element-android#5446]
### Steps to reproduce

1. Send hi-res image and select not to send in original resolution
2. View that sent image
3. Image is shown is original resolution, not the shrinken one as the counterside sees

#### What did you expect?
Image viewer should show the sent image, not the original one

#### What happened instead?
Image viewer shows the original image, thus you cannot check whether the compression is ok for you. You need to download it and check it from downloads.

### Your phone model

Samsung Galaxy S20 FE

### Operating system version

Android 12

### Application version and app store

Element: 1.4.2

### Homeserver

own

### Will you send logs?

No

**Comment by ouchadam** [comment]
when the compression fails we're reusing the original file, they might be a silent failure happening 

looping in product, how should we handle compression errors? 
- some options including showing a dialog/toast to explain the compression failed

similar issue for video #5113

**Comment by waclaw66** [comment]
I don't think that compression fails. The recipient sees shinken image, the image is properly compressed (I can see it on my homeserver's file storage) but sender sees the original hi-res image.

Nevetheless I observed something interresting.
a) if I flush media cache or event whole Element cache from settings and kill the app and start it again. It doesn't help, image is still hi-res.
b) if I rename the original image file in phone storage and flush media cache, the compressed image is shown.

It seems that the original file is shown from phone storage if exists instead of the one from file cache or homeserver.
